types/stylelint/index.d.ts

```diff
@@ -109,6 +109,7 @@ declare namespace stylelint {
 		ignorePatterns?: string;
 		rules?: ConfigRules;
 		quiet?: boolean;
+		formatter?: string;
```

Comment:
[suggest] Probably, to resolve the type errors, this might be required to be:

```suggestion
		formatter?: FormatterType | Formatter;
```

Reply:
Or we could do this:

```suggestion
		formatter?: FormatterType;
```